### PR TITLE
ManifestDread as separate Effect

### PR DESCRIPTION
**Pull request by Hanmac**
+ new ManifestDread Trigger

(old cards needs to be updated)

### CODE REVIEW

File: forge-game/src/main/java/forge/game/ability/effects/ManifestDreadEffect.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import forge.game.ability.AbilityKey;
+import forge.game.card.Card;
+import forge.game.card.CardCollection;
+import forge.game.card.CardZoneTable;
+import forge.game.player.Player;
+import forge.game.spellability.SpellAbility;
+import forge.game.trigger.TriggerType;
+import forge.util.Localizer;
+
+public class ManifestDreadEffect extends ManifestBaseEffect {
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/ability/effects/ManifestDreadEffect.java:15]: mb inherit from ManifestEffect so ``internalEffect`` can be reused too?

**Review by tool4ever — APPROVED** [review]

**Comment by Hanmac** [comment]
@tool4ever might need to add `Paranormal Analyst`

**Comment by tool4ever** [comment]
I thought you wanted me on Door stuff too :p
I'll just mark the card as scriptable for someone imo

File: forge-gui/res/cardsfolder/upcoming/valgavoths_onslaught.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Name:Valgavoth's Onslaught
 ManaCost:X X G
 Types:Instant
-A:SP$ Manifest | Dread$ True | Times$ X | RememberManifested$ True | SubAbility$ DBPutCounterAll
+A:SP$ ManifestDread | Amount$ X | RememberManifested$ True | SubAbility$ DBPutCounterAll
 SVar:DBPutCounterAll:DB$ PutCounterAll | ValidCards$ Card.IsRemembered | CounterType$ P1P1 | CounterNum$ X | SubAbility$ DBCleanup
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/valgavoths_onslaught.txt:5]: @tool4ever wouldn't this be better? `PutCounter | Defined$ RememberedCard` ?

**Pull request merged**